### ISSUE TITLE
Exception generated under Django 1.0.4

### ISSUE THREAD

**Issue by wamcvey** [codysoyland/django-template-repl#1]
Generated traceback when running under django 1.0.4:

```
$ python manage.py templateshell
 Traceback (most recent call last):
   File "manage.py", line 11, in <module>
     execute_manager(settings)
   File "/usr/local/python_environments/hpc_controller/lib/python2.5/site-packages/django/core/management/__init__.py", line 361, in execute_manager
     utility.execute()
   File "/usr/local/python_environments/hpc_controller/lib/python2.5/site-packages/django/core/management/__init__.py", line 306, in execute
     self.fetch_command(subcommand).run_from_argv(self.argv)
   File "/usr/local/python_environments/hpc_controller/lib/python2.5/site-packages/django/core/management/__init__.py", line 260, in fetch_command
     klass = load_command_class(app_name, subcommand)
   File "/usr/local/python_environments/hpc_controller/lib/python2.5/site-packages/django/core/management/__init__.py", line 67, in load_command_class
     {}, {}, ['Command']), 'Command')()
   File "/usr/local/python_environments/hpc_controller/lib/python2.5/site-packages/template_repl/management/commands/templateshell.py", line 2, in <module>
     from django.test.utils import ContextList, setup_test_environment
 ImportError: cannot import name ContextList
 (1)
```

**Comment by wamcvey** [comment]
On a development server with the same code but running 1.1.x, templateshell works fine, so I think the issue is specific to 1.0.x.

**Comment by codysoyland** [comment]
Fixed in 77ba0757b8c50da2434c201eac9c4a28f9035d2d.
